pkg/expr/classic/classic.go

```diff
@@ -203,19 +203,19 @@ func UnmarshalConditionsCmd(rawQuery map[string]interface{}, refID string) (*Con
 		cond := condition{}
 
 		if i > 0 && cj.Operator.Type != "and" && cj.Operator.Type != "or" {
-			return nil, fmt.Errorf("classic condition %v operator must be `and` or `or`", i+1)
+			return nil, fmt.Errorf("condition %v operator must be `and` or `or`", i+1)
 		}
 		cond.Operator = cj.Operator.Type
 
 		if len(cj.Query.Params) == 0 || cj.Query.Params[0] == "" {
-			return nil, fmt.Errorf("classic condition %v is missing the query refID argument", i+1)
+			return nil, fmt.Errorf("condition %v is missing the query refID argument", i+1)
 		}
 
 		cond.QueryRefID = cj.Query.Params[0]
 
 		cond.Reducer = classicReducer(cj.Reducer.Type)
 		if !cond.Reducer.ValidReduceFunc() {
-			return nil, fmt.Errorf("reducer '%v' in condition %v is not a valid reducer", cond.Reducer, i+1)
+			return nil, fmt.Errorf("invalid reducer '%v' in condition %v", cond.Reducer, i+1)
```

Comment:
Should we remove the single quotes here to be consistent with the other errors?

```suggestion
			return nil, fmt.Errorf("invalid reducer %v in condition %v", cond.Reducer, i+1)
```

Reply:
I would rather update all other usages to wrap in single quotes because it would help highlight the name of the function from the rest of the sentence. I generally find it useful to wrap variables in the message.

Reply:
I checked the changes I've made and single quotes are consistently used to highlight variables